src/api/schema/metrics/filter.rs

```diff
@@ -45,6 +48,25 @@ fn sum_metrics_owned<I: IntoIterator<Item = Metric>>(metrics: I) -> Option<Metri
     Some(iter.fold(m, |mut m1, m2| if m1.update(&m2) { m1 } else { m2 }))
 }
 
+#[derive(Default, InputObject)]
+pub struct OutputsFilter {
```

Comment:
Do we still want to drop this?

Reply:
Ah, great catch, I could've sworn I'd removed that in b8d9b8b. Will drop it.